apps/modernization-ui/src/apps/page-builder/components/TargetQuestion/TargetQuestion.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useEffect } from 'react';
+import {
+    ButtonGroup,
+    Checkbox,
+    ModalFooter,
+    ModalRef,
+    ModalToggleButton,
+    Tag,
+    Icon as UswIcon
+} from '@trussworks/react-uswds';
+import { PagesResponse } from 'apps/page-builder/generated';
+import { RefObject, useContext, useState } from 'react';
+import { UserContext } from 'user';
+import { ModalComponent } from 'components/ModalComponent/ModalComponent';
+import './TargetQuestion.scss';
+import { Icon } from '../../../../components/Icon/Icon';
+import { fetchPageDetails } from '../../services/pagesAPI';
+
+type CommonProps = {
+    modalRef: RefObject<ModalRef>;
+    pageId: string;
+    getList?: (data: any) => void;
+    multiSelected?: boolean;
+    header?: string;
+};
+
+type QuestionProps = {
+    id: number;
+    question: string;
+    name: string;
+    selected: boolean;
+    valueSet: string;
+};
+
+const TargetQuestion = ({ modalRef, pageId, getList, header, multiSelected = true }: CommonProps) => {
+    const { state } = useContext(UserContext);
+    const token = `Bearer ${state.getToken()}`;
```

Comment:
Suggestion: There is a convenience method: `authorization()`, that will return the token in the `Bearer <token>` format